analyses/TX_cd_2020/TX_helpers.R

```diff
@@ -0,0 +1,76 @@
+#' Prep Particles for Partial SMC
```

Comment:
Let's move this into the main R folder so it can be shared between analyses, as we are using it for CA (and presumably FL).